library/core/src/main/java/com/google/android/exoplayer2/ExoPlayerImplInternal.java

```diff
@@ -2312,8 +2312,9 @@ private boolean shouldContinueLoading() {
       return false;
     }
     MediaPeriodHolder loadingPeriodHolder = queue.getLoadingPeriod();
+    long bufferedPositionUs = loadingPeriodHolder.getBufferedPositionUs();
     long bufferedDurationUs =
-        getTotalBufferedDurationUs(loadingPeriodHolder.getNextLoadPositionUs());
+        getTotalBufferedDurationUs(bufferedPositionUs);
     long playbackPositionUs =
```

Comment:
One line change to use `getBufferedPositionUs()`.  Local variables is for logging added to commit 78cf217c8f